var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -96,6 +96,7 @@ class Gdal(AutotoolsPackage):
     variant('cryptopp',  default=False, description='Include cryptopp support')
     variant('crypto',    default=False, description='Include crypto (from openssl) support')
     variant('grib',      default=False, description='Include GRIB support')
+    variant('64bitIO',   default=True,  description='Enable 64-bit file I/O')
```

Comment:
Variant name changed as per Adam's discussion. This can be reviewed again and merged, I will bring in any additional updates based on the spack PR https://github.com/spack/spack/pull/30809 afterwards.